isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -278,10 +282,18 @@ namespace Isis {
    *
    * @return @b QString The name of the output control network.
    */
-  void BundleSolutionInfo::setOutputControl(Control *outputControl) {
+  void BundleSolutionInfo::setOutputControl(Control * outputControl) {
     m_outputControl = outputControl;
   }
+    
+  void BundleSolutionInfo::setOutputControlName(QString name) {
+
+            m_outputControlName = name;
```

Comment:
This member variable does not appear to be instantiated anywhere in the class...?

Reply:
Good catch.  It's now instantiated to an empty string in all of the constructors (including the copy constructor).